src/test/java/org/folio/rest/impl/UsersAPIConsortiaTest.java

```diff
@@ -106,7 +106,7 @@ void doNotSendUserCreateUpdateKafkaEventsForShadowUser() {
   }
 
   @Test
-  void canDeleteAUserForConsortia() {
+  void canDeleteStaffAUserForConsortia() {
```

Comment:
please add asserts to UserTenantIT that field 'consortiumId' is populated

Reply:
Added assert to UserTenantIT